### DISCUSSION TITLE
Can't do anything on one of the repos

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#24078] [Apps, API and Webhooks]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/e/bea10152b7906117856faa21af8f1e20422996f2.jpeg" data-download-href="/uploads/short-url/rcnFCJJk4DRRzVdEVtDbkoroxB8.jpeg?dl=1" title="Screenshot_20210512-120147_GitHub" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/6032823/181081828-4c7ae780-0536-4052-90a9-1bd0f497a361.jpeg" alt="Screenshot_20210512-120147_GitHub" data-base62-sha1="rcnFCJJk4DRRzVdEVtDbkoroxB8" width="225" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/e/bea10152b7906117856faa21af8f1e20422996f2_2_225x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/b/e/bea10152b7906117856faa21af8f1e20422996f2_2_337x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/e/bea10152b7906117856faa21af8f1e20422996f2_2_450x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/e/bea10152b7906117856faa21af8f1e20422996f2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20210512-120147_GitHub</span><span class="informations">720×1600 122 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
I get this whenever I try to open Cog-Creators/Red-DiscordBot repository or anything on it like pull request for example:<br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/2/6251a2ed159736070d071856ed375fde6832e5d1.jpeg" data-download-href="/uploads/short-url/e1LHeBLqGyfDEpV1RefuRg9G5Gh.jpeg?dl=1" title="Screenshot_20210512-120314_GitHub" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/6032823/181081829-ec37584e-53bc-4a74-92db-6834e4227440.jpeg" alt="Screenshot_20210512-120314_GitHub" data-base62-sha1="e1LHeBLqGyfDEpV1RefuRg9G5Gh" width="225" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/2/6251a2ed159736070d071856ed375fde6832e5d1_2_225x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/6/2/6251a2ed159736070d071856ed375fde6832e5d1_2_337x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/2/6251a2ed159736070d071856ed375fde6832e5d1_2_450x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/2/6251a2ed159736070d071856ed375fde6832e5d1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20210512-120314_GitHub</span><span class="informations">720×1600 122 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I’m running v1.15.0-beta (10007)</p>

**Comment by arnested** [comment]
<p>I have got the same problem.</p>
<p>I was on the beta as well. I uninstalled, left the beta program and installed the stable version (1.14).</p>
<p>I also tried to remove all storage. The issue persists.</p>

**Comment by Jackenmen** [comment, marked as answer]
<p>It seems that the affected repository works fine on my mobile now (still running the same app version). I’ll report back if this becomes an issue again.</p>

**Comment by arnested** [comment]
<p>Works for me as well now.</p>
<p>Also no app update. Must have been a server side thing.</p>
